great_tables/_tbl_data.py

```diff
@@ -545,3 +556,38 @@ def _(df: PdDataFrame) -> PdDataFrame:
 @validate_frame.register
 def _(df: PlDataFrame) -> PlDataFrame:
     return df
+
+
+# to_frame ----
+
+
+@singledispatch
+def to_frame(ser: "list[Any] | SeriesLike", name: Optional[str] = None) -> DataFrameLike:
```

Comment:
This supports the `vals.fmt_*` functions, by converting a list or series into a DataFrame (which can be fed into GT)